inventory/host_vars/_example.com/config.yml

```diff
@@ -29,3 +29,11 @@ mail_domain: example.com
 #
 #custom_application_env_vars: |
 #  OFN_FEATURE_CONNECT_AND_LEARN: "true"
+
+# Overridable Image Settings
+# If you update attachment_styles, you will need to regenerate thumbnails. Use rake paperclip:refresh:thumbnails CLASS=Spree::Image
```

Comment:
:bomb: I just read this is in the docs:

> WARNING: regeneration is broken on S3 and will truncate your original images.
https://github.com/thoughtbot/paperclip/wiki/Thumbnail-Generation#generatingregenerating-your-thumbnails

That is true for version 6.1.0 while we use version 3.4.2. I don't know when that bug was introduced but the issue was reported nine months after the last release. So we might be safe for a while.

Reply:
it's good to know :+1: